### ISSUE TITLE
Loading word embeddings in the validation phase

### ISSUE THREAD

**Issue by amirj** [dennybritz/chatbot-retrieval#21]
Word embedding is initialized at the beginning. It's tuned during the learning process. When evaluating the model I see the following message in the output:

> INFO:tensorflow:Saving evaluation summary for 17400 step: loss = 1.10763,...
> INFO:tensorflow:No glove/vocab path specificed, starting with random embeddings.
> 

I'm interested to know if the learned embeddings is stored in the saved checkpoints?
If yes, why the above message is appear in the evaluation phase?

**Comment by dennybritz** [comment]
Yes, they are stored. The message appears when the graph is constructed, which happens ever time the evaluation runs.
